### ISSUE TITLE
Set last_modified_date to time of merge

### ISSUE THREAD

**Issue by abdon-copper** [abdon-gadgets/merge-jwl#3]
Currently the time is taken from the first input file, this looks weird because JW Library shows this time in the restore dialog.

![image](https://user-images.githubusercontent.com/59224618/103167044-0da8f380-4828-11eb-9522-9435e67b6789.png)

https://github.com/abdon-gadgets/merge-jwl/blob/403781eff3f90e92532b39d974ad0c629d9df2d4/src/main.rs#L186

**Comment by jay-aware** [comment]
Hi, can I try this?

**Comment by abdon-copper** [comment]
Of course, you are welcome to contribute. This project is not that well documented, so feel free to contact me if you need help with setting up the programming environment or understanding the code structure.
